### ISSUE TITLE
`cargo vet inspect` leaves a stale file lock

### ISSUE THREAD

**Issue by bholley** [mozilla/cargo-vet#149]
STR:
checkout m-c
cargo vet suggest
cargo vet inspect matches 0.1.9
ctrl-D to exit the nested shell
cargo vet suggest

Output:
```
Error: Could not acquire lockfile at /var/folders/.../cargo-vet-checkout/lockfile
```

If I manually remove the lockfile, `cargo vet certify` doesn't seem to remember about matches-0.1.9, so presumably there's some state flush that's not happening.

**Comment by Gankra** [comment]
probably because of pushd but the "fix" is most ideally #146

**Comment by bholley** [comment]
> probably because of pushd

We're not using pushd though? You just mean the nested shell?

> but the "fix" is most ideally #146

if we can get to it within the next day or so — the experience is pretty broken here.

**Comment by bholley** [comment]
I fixed this in #166.
